### PR TITLE
refactor: (Input) CC => FC

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [x] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | refactor: CC => FC |
| 🇨🇳 Chinese | refactor: CC => FC |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/input/TextArea.tsx [hunk 1]
```diff
@@ -224,7 +223,6 @@ const TextArea = React.forwardRef<TextAreaRef, TextAreaProps>(
         value={val}
         element={textArea}
         handleReset={handleReset}
-        ref={clearableInputRef}
```
Comment (li-jia-nan) [inline components/input/TextArea.tsx:227]: 这个 ref 是多余的，这个组件内部没有用到 ref

File: components/input/TextArea.tsx [hunk 2]
```diff
@@ -219,12 +218,10 @@ const TextArea = React.forwardRef<TextAreaRef, TextAreaProps>(
         disabled={mergedDisabled}
         {...props}
         prefixCls={prefixCls}
-        direction={direction}
```
Comment (li-jia-nan) [inline components/input/TextArea.tsx:222]: 这个 direction 直接在里面从 useContext 读取即可，没必要从外面传进去，删掉了

File: components/input/TextArea.tsx (hunk 1 above)
Comment (li-jia-nan) [inline components/input/TextArea.tsx:227]: 这个 ref 是多余的，这个组件内部没有用到 ref，删掉了

**Comment by afc163** [comment]
这个得 @MadCcc 看

**Comment by MadCcc** [comment]
这个组件不改了，要删了，功能已经下沉到 rc-textarea，ref：https://github.com/react-component/textarea/pull/30

**Comment by li-jia-nan** [comment]
看到 Bundle size optimization 的 issues 想起来顺手改了

**Comment by li-jia-nan** [comment]
OK，那就close了

**Pull request closed without merging**